src/registry/check.rs

```diff
@@ -46,6 +46,11 @@ pub struct RegistryCheckArgs {
     #[arg(long, default_value = "false")]
     pub display_policy_coverage: bool,
 
+    /// Enable the most recent validation rules for the semconv registry. It is recommended
+    /// to enable this flag when checking a new registry.
```

Comment:
Maybe add a note that the `semantic_conventions` main branch should always enable this flag?

Reply:
Comment updated.